### PR TITLE
Non user styling

### PR DESCRIPTION
**Pull request by NTR-TheMekaVerse**

### CODE REVIEW

File: app/javascript/components/components/Header.js [hunk 1]
```diff
@@ -23,30 +17,39 @@ class Header extends Component {
...
+              <NavLink to={homeLink}>
+                <img id="eyeLogo" src={eyeLogo} alt="home link" />
+              </NavLink>
+              
+            </Col>
+
+           
             <Nav className="me-auto" navbar>
-              <NavItem>
+    <div className="headerLinks">
```
Comment (elysemontano) [inline app/javascript/components/components/Header.js:33]: Indentation looks a little off here.

**Review by elysemontano — CHANGES_REQUESTED** [review]
A few small adjustments to look at.

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -23,30 +17,39 @@ class Header extends Component {
...
-              </NavItem>
-
-              <NavItem>
-                <NavLink to={homeLink}>
-                  <img src={home_image} alt="home link" />
-                </NavLink>
-              </NavItem>
+                </NavItem>
 
+{' '}
```
[the remark this answers is not in the document]
Comment (nataliadiaz2) [reply app/javascript/components/components/Header.js:38]: This is an alternative to the break tag Zan mentioned to us.

**Review by elysemontano — APPROVED** [review]
Looks good!

**Pull request merged**